api/clients/verification/verification_utils_test.go

```diff
@@ -0,0 +1,141 @@
+	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"os"
+	"runtime"
+	"testing"
+)
+
+var (
+	gettysburgAddressBytes = codec.ConvertByPaddingEmptyByte(
+		[]byte("Fourscore and seven years ago our fathers brought forth, on this continent, a new nation, conceived in liberty, and dedicated to the proposition that all men are created equal. Now we are engaged in a great civil war, testing whether that nation, or any nation so conceived, and so dedicated, can long endure. We are met on a great battle-field of that war. We have come to dedicate a portion of that field, as a final resting-place for those who here gave their lives, that that nation might live. It is altogether fitting and proper that we should do this. But, in a larger sense, we cannot dedicate, we cannot consecrate—we cannot hallow—this ground. The brave men, living and dead, who struggled here, have consecrated it far above our poor power to add or detract. The world will little note, nor long remember what we say here, but it can never forget what they did here. It is for us the living, rather, to be dedicated here to the unfinished work which they who fought here have thus far so nobly advanced. It is rather for us to be here dedicated to the great task remaining before us—that from these honored dead we take increased devotion to that cause for which they here gave the last full measure of devotion—that we here highly resolve that these dead shall not have died in vain—that this nation, under God, shall have a new birth of freedom, and that government of the people, by the people, for the people, shall not perish from the earth."))
+
+	kzgConfig       *kzg.KzgConfig
+	kzgVerifier     *verifier.Verifier
+	testRandom      *random.TestRandom
+	srsNumberToLoad uint64
+)
+
+func setup() {
+	log.Println("Setting up suite")
+
+	srsNumberToLoad = 2900
+
+	kzgConfig = &kzg.KzgConfig{
+		G1Path:          "../../../inabox/resources/kzg/g1.point",
+		G2Path:          "../../../inabox/resources/kzg/g2.point",
+		G2PowerOf2Path:  "../../../inabox/resources/kzg/g2.point.powerOf2",
+		CacheDir:        "../../../inabox/resources/kzg/SRSTables",
+		SRSOrder:        3000,
+		SRSNumberToLoad: srsNumberToLoad,
+		NumWorker:       uint64(runtime.GOMAXPROCS(0)),
+		LoadG2Points:    false,
+	}
+
+	testRandom = random.NewTestRandom()
+}
+
+// randomlyModifyBytes picks a random byte from the input array, and increments it
+func randomlyModifyBytes(inputBytes []byte) {
+	indexToModify := testRandom.Intn(len(inputBytes))
+	inputBytes[indexToModify] = inputBytes[indexToModify] + 1
+}
+
+func TestMain(m *testing.M) {
+	setup()
+	result := m.Run()
+	teardown()
+	os.Exit(result)
+}
+
+func teardown() {
+	log.Println("Tearing down")
+}
+
+func TestComputeAndCompareKzgCommitmentSuccess(t *testing.T) {
+	kzgVerifier, err := verifier.NewVerifier(kzgConfig, nil)
+	assert.NotNil(t, kzgVerifier)
+	assert.Nil(t, err)
```

Comment:
Personally, I prefer using `require`. Assert will allow the remainder of the test to run and can sometimes permit odd errors. `require` causes the test to abort immediately when the first failure is detected.
```suggestion
	require.NotNil(t, kzgVerifier)
	require.Nil(t, err)
```

Reply:
Done